package.json

```diff
@@ -121,5 +121,7 @@
     "typecheck": "tsc --build src/tsconfig.json",
     "report-coverage": "codecov -f coverage/coverage-final.json",
     "version": "make clean build"
-  }
+  },
+  "private": true,
```

Comment:
Setting `"private": true` here will I believe prevent the root package from being published to npm during the release process. We're going to need to figure out how we'll continue to build and publish the Hypothesis client following these changes.

Reply:
> What caused the changes in `yarn.lock` on this branch?

That may have been prev changes that I did, but forgot to revert the lock. Let me revert that file and add back the workspace name and see if the lock changes after an install.